inception-external-search-elastic/src/main/java/de/tudarmstadt/ukp/inception/externalsearch/elastic/ElasticSearchProvider.java

```diff
@@ -92,15 +94,24 @@ public boolean isConnected()
         remoteUrl = properties.getRemoteUrl();
         indexName = properties.getIndexName();
         searchPath = properties.getSearchPath();
+        randomOrder = properties.isRandomOrder();
         
         // Set headers
         HttpHeaders headers = new HttpHeaders();
         headers.setContentType(MediaType.APPLICATION_JSON);
 
         
         // Set query
-        String query = "{\"size\":%d,\"query\":{\"match\":"
+        String query;
+        if (randomOrder) {
```

Comment:
Can we build this up in a structured way and convert it to JSON? Right now there is significant repetition of the JSON request content between the two branches.

Reply:
Also, a comment to document what is going on here would be nice.  You might want to give a name like `ARBITRARY_FIXED_SEED` to `5` to make clear there is nothing magic about it.

Reply:
Maybe the seed should be a trait property too?

Reply:
It could be - it looks like it is easy to add properties, so let's go ahead and do that @shinyichen

Reply:
Would you let use edit the seed in the config?

Reply:
There's probably little harm in it, except perhaps confusion.  Is it easy to provide some sort of help string for a config setting?

Reply:
Tooltip?

Reply:
I can look up how it's done.